app/database_etl/summary_report_generator/summary_report.py

```diff
@@ -0,0 +1,64 @@
+from time import time
+from app.serotracker_sqlalchemy import db_session, DashboardSource, ResearchSource, \
+    Country, State, City, TestManufacturer, AntibodyTarget, CityBridge, StateBridge, \
+    TestManufacturerBridge, AntibodyTargetBridge
+from sqlalchemy import func, inspect
+from app.utils.notifications_sender import send_slack_message
+import json
+import datetime
+
+class SummaryReport:
+    def __init__(self):
+        self.start_time = time()
+        self.get_table_counts_before()
+
+    def get_elapsed_time(self):
+        return time() - self.start_time
+
+    def get_num_airtable_records(self, num_records: int):
+        self.num_airtable_records = num_records
+
+    def get_table_counts(self):
+        table_counts = {}
+        with db_session() as session:
+            for table in [DashboardSource, ResearchSource, Country, State, City,
+                          TestManufacturer, AntibodyTarget, CityBridge, StateBridge,
+                          TestManufacturerBridge, AntibodyTargetBridge]:
+                # Inspect primary key of table
+                pk = inspect(table).primary_key[0].key
+                # Use primary key to get row count instead of using the .count() method for perf reasons
+                table_counts[table.__tablename__] = session.query(func.count(getattr(table, pk))).scalar()
```

Comment:
does this work instead? `select([func.count()]).select_from(table)`? reference: https://stackoverflow.com/questions/12941416/how-to-count-rows-with-select-count-with-sqlalchemy

Reply:
it does but i am more down to use ORM layer, more consistent with the rest of our sqlalchemy usage